clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1752,6 +1752,7 @@ class SyclKernelBodyCreator : public SyclKernelFieldHandler {
   const CXXRecordDecl *KernelObj;
   llvm::SmallVector<Expr *, 16> MemberExprBases;
   FunctionDecl *KernelCallerFunc;
+  SourceLocation KCFSL; // KernelCallerFunc source location.
```

Comment:
Please use a more descriptive name. Something like KernelCallerSrcLoc

Reply:
Thanks for the suggestion. Fixed.